build/int.cloudbuild.yaml

```diff
@@ -26,16 +26,22 @@ steps:
   - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
   - 'TF_VAR_group_org_admins=test-gcp-org-admins@test.blueprints.joonix.net'
   - 'TF_VAR_group_billing_admins=test-gcp-billing-admins@test.blueprints.joonix.net'
+  - 'TF_VAR_im_github_pat_secret=$_IM_GITHUB_PAT_SECRET_ID'
+  secretEnv: ['IM_GITHUB_PAT']
 - id: init-all
   waitFor:
     - prepare
   name: 'gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS'
   args: ['/bin/bash', '-c', 'cft test run all --stage init --verbose']
+  secretEnv: ['IM_GITHUB_PAT']
+  env:
+  - 'TF_VAR_im_github_pat_secret=$_IM_GITHUB_PAT_SECRET_ID'
```

Comment:
nit: I think this is unnecessary

Reply:
Currently following the [example here](https://github.com/GoogleCloudPlatform/terraform-google-tf-cloud-agents/blob/main/build/int.cloudbuild.yaml#L184-L187) which sets up a similar secret to be used here [https://github.com/GoogleCloudPlatform/terraform-google-tf-cloud-agents/blob/main/build/int.cloudbuild.yaml#L156](https://github.com/GoogleCloudPlatform/terraform-google-tf-cloud-agents/blob/main/build/int.cloudbuild.yaml#L156).

Maybe the other way of explicitly creating the TF_VAR link could work, just involves some other changes I don't recall at the top of my head.

Reply:
In this case it was just a nit that this init step wont care about the env var so we could skip it